cpp/src/bitmask/null_mask.cu

```diff
@@ -412,6 +412,27 @@ void concatenate_masks(std::vector<column_view> const &views,
     dest_mask, number_of_mask_bits);
 }
 
+// set bits [0, false_begin_bit) as true, set bits [false_begin_bit, size) as false
+void clear_bits_from(bitmask_type* mask, size_type size,
```

Comment:
Let's rename this and generalize it: I think it should be `set_bit_range()` and take a begin and end index, and a bool for whether to clear or set the bits.

```
void set_bit_range(bitmask_type* mask, size_type begin, size_type end, bool value, cudaStream_t stream)
```

Reply:
I could not think of appropriate short name for this (partition_bit_range or set_bits_before_and_clear_bits_after)
This function `clear_bits_from` just do writes only. I thought about `set_bit_range`, but it needs to **read the device data** and then do cudaMemSet. 
should I change the functionality of this function to `set_bit_range` or rename this function that describes it clearly?

Reply:
@harrism

Reply:
The naming difficulty comes from the fact that this function does more than "One Thing". 

Instead, `set_null_mask` should be updated to take a `begin/end` range: https://github.com/rapidsai/cudf/blob/f5b0a5d92f289ab3d1e01e683702c1020913968e/cpp/include/cudf/null_mask.hpp#L92

Then your use case can be accomplished via two calls to `set_null_mask`.

Reply:
I like the approach taken. Resolving.